_docs/movement/physical_jump.md

```diff
@@ -22,12 +22,12 @@ The functionality works out of the box but can be further configured:
 
 ### XRToolsMovementPhysicalJump
 
-| Property              | Description                                                     |
-| --------------------- | --------------------------------------------------------------- |
-| Enabled               | When ticked the movement function is enabled.                   |
-| Order                 | The order in which this movement is applied when multiple movement functions are used.  |
-| Body Jump Enable      | Enables detection of the player physical jump.  |
-| Body Jump Player Only | If enabled, the ground-physics settings are ignored and the player jumps as high as their real-world jump. |
-| Body Jump Threshold   | Adjustment for physical jump detection. |
-| Arms Jump Enable      | Enables detection of jump by swinging arms up. |
-| Arms Jump Threshold   | Adjustment for arm jump detection. | 
+| Property | Description |
+| ---- | ------------ |
+| Enabled               | When ticked the movement function is enabled |
+| Order                 | The order in which this movement is applied when multiple movement functions are used |
+| Body Jump Enable      | Enables detection of the player physical jump |
+| Body Jump Player Only | If enabled, the ground-physics settings are ignored and the player jumps as high as their real-world jump |
+| Body Jump Threshold   | Adjustment for physical jump detection |
+| Arms Jump Enable      | Enables detection of jump by swinging arms up |
+| Arms Jump Threshold   | Adjustment for arm jump detection | 
```

Comment:
Just mentioning here but this is broader and I think Godot has a feature for this now, but we should add in unit type. Is this a velocity (m/s) or a force (N)....

Reply:
I'm not seeing an option to add units to the export, but I added units to the description here in the documentation.